client/Lykke.Service.Subscribers.Client/AutofacExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Autofac;
+using Common.Log;
+
+namespace Lykke.Service.Subscribers.Client
+{
+    public static class AutofacExtension
+    {
+        public static void RegisterInspireStreamClient(this ContainerBuilder builder, string serviceUrl, ILog log, int timeOut)
+        {
+            if (builder == null) throw new ArgumentNullException(nameof(builder));
+            if (serviceUrl == null) throw new ArgumentNullException(nameof(serviceUrl));
+            if (log == null) throw new ArgumentNullException(nameof(log));
+            if (string.IsNullOrWhiteSpace(serviceUrl))
+                throw new ArgumentException("Value cannot be null or whitespace.", nameof(serviceUrl));
+
+            builder.RegisterInstance(new SubscribersClient(serviceUrl, log, timeOut)).As<SubscribersClient>().SingleInstance();
```

Comment:
`.As<SubscribersClient> -> .As<ISubscribersClient>`

Reply:
I've already removed this class? where do you see it now?

Reply:
maybe some of the changes are not pushed correctly, I will review it again!